rlog/2024-04-20-rln-light-verifiers.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: 'Verifying RLN proofs in light clients with Subtrees'
```

Comment:
Change to titlecase for consistency with rest of research blog:
Verifying RLN Proofs in Light Clients with Subtrees

Reply:
addressed in f983bad